### DISCUSSION TITLE
Cannot edit image dialogs in email templates/workflows

### DISCUSSION THREAD

**Discussion by TonyM15** [YetiForceCompany/YetiForceCRM#15163] [General] [answered]
Suddenly it happen that I cannot edit image properties in dialog (templates/workflows)
Cannot manipulate with width/height etc. System has been updated to latest version. 
Same story with cells/tables. Any clues on that?

Cheers

**Comment by mariuszkrzaczkowski** [comment]
record a video showing the problem

**Comment by vovpff** [comment, marked as answer]
Try in incognito mode?

**Reply by TonyM15** [reply]
Cache! Thanks vovpff!

**Comment by s2b-git** [comment]
Here it is not working too. Cache cleared, tried in incognito mode and on various OS's (Ubuntu, Win Server, Win 10)
What it could be?

Best

**Reply by mariuszkrzaczkowski** [reply]
give more details and screenshot,
so that you know where exactly the problem is
